### ISSUE TITLE
Bug: impossible to run without conflict with peerDeps

### ISSUE THREAD

**Issue by valerii15298** [MichalLytek/typegraphql-nestjs#41]
These are peerDeps for typegraphql-nestj:
```
"peerDependencies": {
    "@apollo/federation": "^0.29.0",
    "@apollo/gateway": "^0.38.0",
    "@nestjs/common": "^8.0.6",
    "@nestjs/core": "^8.0.6",
    "@nestjs/graphql": "^8.0.2",
    "apollo-graphql": "^0.9.3",
    "graphql-tag": "^2.12.5",
    "type-graphql": "^1.1.1"
  }
```
I installed them. Notice this:
```
    "@apollo/federation": "^0.29.0",
    "@apollo/gateway": "^0.38.0",
```
But `@nestjs/graphql` requires these:
```
@nestjs/graphql 8.0.2
  ├── ✕ unmet peer @apollo/gateway@"^0.29.0 || ^0.32.0 || ^0.33.0": found 0.38.2
  └── ✕ unmet peer @apollo/federation@"^0.22.0 || ^0.25.2 || ^0.26.0": found 0.29.1
```
So basically there is no way to run typegraphql-nestjs without conflict with dependencies.

Is there some workaround for it? Thanks in advance!
@MichalLytek

**Comment by MichalLytek** [comment]
I need to bump all the deps. A few months ago, I've adapted the codebase to NestJS v7 but the work is not finished on my local branch.

**Comment by MichalLytek** [comment]
Closing as fixed by 3fc0038 🔒
